src/containers/Tenant/Query/QueryResult/components/SimplifiedPlan/SimplifiedPlan.scss

```diff
@@ -15,7 +15,7 @@
         border-collapse: collapse;
         tr {
             &:hover {
-                background-color: var(--g-color-base-generic-hover) !important;
+                background-color: var(--g-color-base-simple-hover) !important;
```

Comment:
Can you change color for other tables as well? Or new color is needed only for this table?
- `PaginatedTables.scss`
- `TabletStorageInfo.scss`

Reply:
Definitely should be done, thank you!